controllers/api/v1/keyword.go

```diff
@@ -0,0 +1,49 @@
+package apiv1
+
+import (
+	"google-scraper/models"
+	"google-scraper/serializers"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/beego/beego/v2/server/web"
+	"github.com/beego/beego/v2/adapter/context"
+	"github.com/beego/beego/v2/adapter/utils/pagination"
+)
+
+var sizePerPage int
+
+type Keyword struct {
+	baseAPIController
+}
+
+func init() {
+	var err error
+	sizePerPage, err = web.AppConfig.Int("sizePerPage")
+	if err != nil {
+		logs.Error("Retrieving sizePerPage failed: ", err)
+	}
+}
+
+func (c *Keyword) Index() {
+	keyword := c.GetString("keyword")
+	keywords := models.GetQuerySeterKeywords(c.CurrentUser, keyword)
+
+	keywordsCount, err := keywords.Count()
+	if err != nil {
+		logs.Error("Retrieving keyword count failed: ", err)
```

Comment:
@junan In case that we have an error here, should we handle something more here.
because if we just only log an error, it still can process to the next step and block the application 🤔